starfish/stack.py

```diff
@@ -119,23 +119,26 @@ def __init__(self, image_partition: TileSet) -> None:
             c = tile.indices[Indices.CH]
             zlayer = tile.indices.get(Indices.Z, 0)
             data = tile.numpy_array
-            if max_size != data.dtype.itemsize:
```

Comment:
I would put this back.  It's a very weird situation and probably should flag it.